test/helpers/integration.rb

```diff
@@ -10,30 +10,30 @@ class TestIntegration < Minitest::Test
   TOKEN = "xxyyzz"
   WORKERS = 2
 
-  BASE = defined?(Bundler) ? "bundle exec #{Gem.ruby} -Ilib" :
-    "#{Gem.ruby} -Ilib"
+  BASE = "#{Gem.ruby} -rrubygems -Ilib"
```

Comment:
This might break tests for people running locally. In CI it will be fine of course. Not sure if we can do this.